global/scripts/FieldMapPreprocessingAll.sh

```diff
@@ -141,18 +180,29 @@ case $DistortionCorrection in
 
     ${GENERAL_ELECTRIC_METHOD_OPT})
 
-        # -----------------------------------------------
-        # -- General Electric Gradient Echo Field Maps --
-        # ----------------------------------------------- 
+        # ---------------------------------------------------
+        # -- GE HealthCare Legacy Gradient Echo Field Maps --
+        # --------------------------------------------------- 
 
-        ${FSLDIR}/bin/fslsplit ${GEB0InputName}     # split image into vol0000=fieldmap and vol0001=magnitude
-		    mv vol0000.nii.gz ${WD}/FieldMap_deg.nii.gz
-		    mv vol0001.nii.gz ${WD}/Magnitude.nii.gz
+            ${FSLDIR}/bin/fslsplit ${GEB0InputName}     # split image into vol0000=fieldmap and vol0001=magnitude
+		    ${FSLDIR}/bin/immv vol0000 ${WD}/FieldMapHertz
+		    ${FSLDIR}/bin/immv vol0001 ${WD}/Magnitude
 		    ${FSLDIR}/bin/bet ${WD}/Magnitude ${WD}/Magnitude_brain -f 0.35 -m #Brain extract the magnitude image
-		    ${FSLDIR}/bin/fslmaths ${WD}/FieldMap_deg.nii.gz -mul 6.28 ${WD}/FieldMap.nii.gz
+		    ${FSLDIR}/bin/fsl_prepare_fieldmap GEHC_FIELDMAPHZ ${WD}/FieldMapHertz ${WD}/Magnitude_brain ${WD}/FieldMap ${DeltaTE}
```

Comment:
This has some tabs vs spaces going on, and since you are editing these lines anyway (so git blame will show this PR), can you clean that up?  Personally, I prefer 4 spaces per level, but we don't really have a consensus (other than "consecutive lines should use the same convention").

Reply:
Not sure what happen here. I normally use 4 space as well.